dvc/command/experiments.py

```diff
@@ -525,6 +557,10 @@ def run(self):
             fill_value = "" if self.args.csv else FILL_VALUE
             iso = True if self.args.csv else False
 
+            if self.args.html:
+                self.args.only_changed = True
```

Comment:
As this logic always goes with html, I guess it is better to move them into `show_experiments` or `show_experiments` will go wrong if it was called elsewhere without them.

Reply:
I tried to follow the logic added for `csv` above (i.e. set `iso = True`) which modifies arguments for `show_experiments`.

`show_experiments` won't really fail if these args have different values, these are just defaults that I assume to make the plot cleaner.

Reply:
The logic here is a bit complex. Any better ways to handle them? @skshetry

Reply:
I moved the logic to inside `show_experiments`